webapp/src/components/BuyEstatePage/BuyEstatePage.js

```diff
@@ -116,7 +113,11 @@ export default class BuyEstatePage extends React.PureComponent {
     const { isConnected, isLoading } = this.props
 
     if (isLoading) {
-      return <Loading />
+      return (
```

Comment:
Hmmm, Do you remember why you need to change it?

It would be great to just use `<Loading />` to show the spinner to avoid code duplication between components

Reply:
My reasoning behind removing this is that this component looked like accidental duplication. It's only function was enabling `<Loader />` for `display: flex` parents.

That said, I can see that we're repeating:

`<div><Loader /></div>`

in quite a few places. I'm not sure if we should abstract this or not, if we do, I'd do it top level, but Loading sounds weird when we do use `Loader` in a couple of places. Once again @cazala  what do you think?